docs/ingest-pipelines.asciidoc

```diff
@@ -1,21 +1,92 @@
+//////////////////////////////////////////////////////////////////////////
+// This content is reused in the Legacy ingest pipeline
+//////////////////////////////////////////////////////////////////////////
+
 [[ingest-pipelines]]
 === Parse data using ingest pipelines
 
-The APM integration loads default ingest node pipelines into {es}.
-These pipelines preprocess and enrich APM documents before indexing them.
+:append-legacy:
+// tag::ingest-pipelines[]
+
+Ingest pipelines preprocess and enrich APM documents before indexing them.
 For example, a pipeline might define one processor that removes a field, and another that renames a field.
+This can be useful for ensuring data security by removing or obfuscating sensitive information.
+See <<apm-data-security,data security>> for more on this topic.
 
-Pipelines can be used to ensure data security by removing or obfuscating sensitive information.
-See <<apm-data-security,data security>> for an example.
+The default APM pipelines are defined in index templates that {fleet} loads into {es}.
+{es} then uses the index pattern in these index templates to match pipelines to APM data streams.
 
-[float]
-[[view-edit-default-pipelines]]
-=== View or edit a default pipeline
+[discrete]
+[id="default-pipeline{append-legacy}"]
+=== Default ingest pipelines
+
+// QQ: Do we want to doc these still?
+// Should we doc them on a per data-stream basis?
```

Comment:
Default ingest pipelines: should we doc which pipelines apply each stream?

Reply:
That's a good point—they are an implementation detail. Removed in 21892a6. We can always add them back in a later PR if Silvia feels strongly otherwise.

Reply:
SGTM